syserol/figures/figure1.py

```diff
@@ -15,11 +15,14 @@ def makeFigure():
 
     cube, glyCube = createCube()
     comps = np.arange(1, 13)
-    arr = [perform_CMTF(cube, glyCube, i)[2] for i in comps]
+    tensorArr = [perform_CMTF(cube, glyCube, i)[2] for i in comps]
```

Comment:
You don't want to do this, as you're running the same factorization twice. Write a loop to save both of these from one factorization run.